docs/source/configuration.rst

```diff
@@ -416,11 +418,18 @@ BLAST configuration options
     [blast]
     db = gs://my-database-bucket/mydatabase
 
-
 **Tip**: to upload your BLAST database to a cloud bucket, please refer to the
 cloud vendor documentation (`AWS <https://docs.aws.amazon.com/AmazonS3/latest/user-guide/upload-objects.html>`_
 or `GCP <https://cloud.google.com/storage/docs/uploading-objects>`_).
 
+If you have BLAST+ available in your machine, you can select the appropriate cloud vendor (``gcp`` or ``aws``) and
+run the command below to get a list of BLAST databases provided by NCBI:
+
+.. code-block:: bash
+
+    update_blastdb.pl --source [gcp|aws] --showall pretty
```

Comment:
The code blocks have a copy feature, so I think one should be able to copy/paste the code block and it should work. So we need an example with either AWS or GCP.

Reply:
Thanks for the feedback, this is done.